packages/semcom-sdk/lib/component/services/register-component.service.spec.ts

```diff
@@ -96,4 +113,116 @@ describe('RegisterComponentService', () => {
 
   });
 
+  it('should generate a correct tag', async () => {
+
+    mockEvalAndDefine();
+
+    const mockComponent = {
```

Comment:
This mockComponent is used in almost every test. Put it as a constant at the top of the file. If you're making changes to it, add it as a global let variable and set it back to its original value in the beforeEach before every test.